src/components/UserProfile/PropertyCarousel.js

```diff
@@ -35,10 +35,15 @@ export default function PropertyCarousel({ properties, title }) {
 				<Carousel navButtonsAlwaysVisible={true} autoPlay={false} animation="slide">
 					{
 						groupsOf3
-							.map((items) => <Stack key={"group " + items[0] + title} direction="row"
+							.map((items) => <Grid container columns={3} key={"group " + items[0] + title} direction="row" spacing={3}
 								alignItems="center" justifyContent="space-evenly">
-								{items.map((x) => <Typography key={"prop " + x + title}>{x}</Typography>)}
-							</Stack>)
+								{items.map((x) => <Grid xs={1} key={x.uid}>
+									<Card sx={{ height: 380 }}>
```

Comment:
Is changing the height here going to reduce the white space for these cards? It would be great to remove those space which were for the like button.

Reply:
oh yeah, i'll definitely look into that now that you brought it up. I just used the exact same style, but you're totally right that's just whitespace now